### PR TITLE
World properties, "chat" always emits even with prefix

### PR DESCRIPTION
**Pull request by ConnorMcGehee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   "license": "ISC",
   "dependencies": {
     "@types/node": "^20.12.7",
+    "dotenv": "^16.4.5",
```
Comment (Anatoly03) [inline package.json:16]: This is not supposed to be part of the API itself.

File: src/events.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export default (client: Client) => {
     client.raw.once('init', async ([id, cuid, username, face, isAdmin, x, y, can_edit, can_god, title, plays, owner, global_switch_states, width, height, buffer]) => {
         await client.send(Magic(0x6B), Bit7(MessageType['init']))
 
-        client.world = new World(width, height)
+        client.world = new World(id, cuid, username, face, isAdmin, x, y, can_edit, can_god, title, plays, owner, global_switch_states, width, height)
```
Comment (Anatoly03) [inline src/events.ts:18]: World is not supposed to track this information.

**Review by Anatoly03 — CHANGES_REQUESTED** [review]
Change of world properties is questionable.

**Comment by ConnorMcGehee** [comment]
Oops ignore the dotenv

I do think world properties should be tracked though, where would you say is better?

**Comment by Anatoly03** [comment]
If you want, you can provide `World` with a public variable`: Player` and set it upon init in events.

Edit: global_switch_states can be kept as an optional parameter maybe?

**Comment by Anatoly03** [comment]
If you want, you can also wait a bit, I’m currently in the process of rebuilding the internal structure of the API in regards to `World`. We could discuss it on Discord and put our effort together.

File: package.json (hunk 1 above)
Comment (Anatoly03) [inline package.json:16]:
```suggestion
```

**Comment by Anatoly03** [comment]
Sorry for the conflicts 😢

**Comment by Anatoly03** [comment]
Chat added in https://github.com/Anatoly03/pixelwalker.js/pull/16

**Pull request closed without merging**